### DISCUSSION TITLE
Unable to navigate to the form to apply with student ID

### DISCUSSION THREAD

**Discussion by KAI-531** [community/community#147465] [GitHub Education]
### Hi everyone,

Hi.
I tried to submit a new application, but when I selected a school and clicked the “Next” button, I got an error message saying “We require applicants to use a school-issued email address to apply."
The first time I applied, I did not have a school-issued email address and applied with my student ID on the next form, but this time I am getting an error message before the form to submit my student ID that says I need to use a school-issued email address, and I cannot proceed.
How can I apply with my student ID?

**Comment by samus-aran** [comment]
Hi @KAI-531 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [What types of evidence does the Student Developer Pack application accept?](https://github.com/orgs/community/discussions/111352#what-types-of-evidence-does-the-student-developer-pack-application-accept)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by KAI-531** [reply]
Thank you!
I will try it again.
